src/main/java/core/basesyntax/validator/length/ValidateLineLengthImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.validator.length;
+
+public class ValidateLineLengthImpl implements ValidateLineLength {
+    private static final int CORRECT_LENGTH = 3;
+    @Override
+    public void isLengthCorrect(String[] line, int lineNumber) {
+        try {
+            if (line.length != CORRECT_LENGTH) {
+                throw new LineHasNotThreeWords();
+            }
+        } catch (LineHasNotThreeWords e) {
```

Comment:
same comment for all your validators - don't throw exceptions just to catch them